content/reference/user/private_locations/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Private Locations"
+description: "Host Load Generators on your private AWS account"
+lead: "Private Locations on your AWS account"
+date: 2021-11-07T14:29:04+00:00
+lastmod: 2021-11-07T14:29:04+00:00
+weight: 21070
+---
+
+A location describe where to start load generators:
+- Public locations are available by default, and listed by regions.
+- Private locations must be configured, and allow you to deploy load generators in your infrastructure.
+
+{{< alert tip >}}
+Private locations allow you to load test in a private infrastructure, without any inbound connection or credential sharing.
+To access this feature, please contact our [technical support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8/group/12/create/59?summary=Private+Locations&description=Contact%20email%3A%20%3Cemail%3E%0A%0AHello%2C%20we%20would%20like%20to%20enable%20the%20private%20locations%20feature%20on%20our%20organization.).
+{{< /alert >}}
+
+## Introduction
+
+To configure private location, you must configure an agent in your infrastructure, called a control plane.
+
+This control plane will be in charge of spawning load generators on different locations, based on the configuration you'll provide.
+
+**TO BE UPDATED!**
+{{< img src="schema.jpg" alt="Infrastructure schema" >}}
+
+The control plane will pull periodically from our API gateway to know if there's any run to handle that requires one (or more) locations of this control plane.
+
+If so, it will start instances (based on the locations configurations) and start the simulation run on them. 
+Those instances will send stats through the API gateway as well.
+
+{{< alert info >}}
+Only two outbounds domain must be allowed in your network:
+- The API Gateway: `api.gatling.io` on port `443`
+- Download of the Gatling simulation on S3: `eu-west-3.amazonaws.com` on port `443`
+{{< /alert >}}
+
+## Control plane configuration
+
+### Control plane token
+
+Access the private locations section by clicking on the Private locations in the navigation bar (only visible if the feature is activated on your organization).
```

Comment:
Add a visual of the pictogram  ? 
<img width="44" alt="image" src="https://user-images.githubusercontent.com/24846829/200796337-09ac9fa2-6935-4ed2-8ecf-671de6b8793a.png">

I have been told to be very specific in the doc, and given the experience with a denominated Mark Webster, I tend to believe it.

Reply:
We don't have those custom icons in the documentation...

Reply:
Just an image as the rest of the screenshots ?

Reply:
@franciscotco The icon library?

Reply:
The icon library is not available for the documentation, do we need a specific screenshot?